images/hybrid/Dockerfile

```diff
@@ -1,18 +1,21 @@
 FROM debian:buster-slim
+ARG TARGETARCH
 
-LABEL Description="Tyk Gateway docker image" Vendor="Tyk"
+LABEL Description="Tyk Hybrid Gateway image" Vendor="Tyk"
 
 RUN apt-get update \
  && apt-get dist-upgrade -y --no-install-recommends redis-server nginx \
-            build-essential python3-setuptools libpython3.7 curl ca-certificates  \
+            python3-setuptools libpython3.7 python3.7-dev curl ca-certificates  \
  && curl https://bootstrap.pypa.io/get-pip.py | python3 \
- && pip3 install grpcio==1.24.0 \
- && apt-get purge -y build-essential \
+ && pip3 install --only-binary ":all:" grpcio \
```

Comment:
This `pip` command seems to be different from the one in the standard image, any reason? Also better to explicitly install `protobuf` too I think.

Reply:
Thank you for catching this!